cmd/virt-handler/virt-handler.go

```diff
@@ -286,6 +290,11 @@ func (app *virtHandlerApp) Run() {
 		0,
 		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
 
+	lwPVC := cache.NewListWatchFromClient(app.virtCli.RestClient(), "persistentvolumeclaims", k8sv1.NamespaceAll, fields.Everything())
+	pvcInformer := cache.NewSharedIndexInformer(lwPVC, &k8sv1.PersistentVolumeClaim{}, app.PVCWatchTimeoutDuration, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
why not use the informers in `pkg/controller`?  file already calls `NewKubeInformerFactory`